spec/src/modules/browse.js

```diff
@@ -634,18 +634,24 @@ describe('ConstructorIO - Browse', () => {
       browse.getBrowseResultsByItemIds(ids);
     });
 
-    it('Should be rejected when invalid ids is provided', () => {
+    it('Should be rejected when invalid ids are provided', () => {
```

Comment:
ids technically 'is' an array. That's why I kept it as is but this reads better.